src/low.rs

```diff
@@ -130,18 +130,23 @@ pub use raw::{ffi_abi, ffi_abi_FFI_DEFAULT_ABI, ffi_cif, ffi_closure, ffi_status
 /// becomes `low::types::void`.
 pub mod types {
     pub use crate::raw::{
-        ffi_type_double as double, ffi_type_float as float, ffi_type_longdouble as longdouble,
-        ffi_type_pointer as pointer, ffi_type_sint16 as sint16, ffi_type_sint32 as sint32,
-        ffi_type_sint64 as sint64, ffi_type_sint8 as sint8, ffi_type_uint16 as uint16,
-        ffi_type_uint32 as uint32, ffi_type_uint64 as uint64, ffi_type_uint8 as uint8,
-        ffi_type_void as void,
+        ffi_type_double as double, ffi_type_float as float, ffi_type_pointer as pointer,
+        ffi_type_sint16 as sint16, ffi_type_sint32 as sint32, ffi_type_sint64 as sint64,
+        ffi_type_sint8 as sint8, ffi_type_uint16 as uint16, ffi_type_uint32 as uint32,
+        ffi_type_uint64 as uint64, ffi_type_uint8 as uint8, ffi_type_void as void,
```

Comment:
@timfish could you format this code with rustfmt if not done so already? I think this would make this block a bit easier to read.

Reply:
I think this was actually run through rust fmt but I'll double check!

Reply:
Yes, this is the output from rustfmt. I even tried it with nightly and get the same result!

Reply:
@timfish Ah perfect, thanks for clarifying.